pkg/scheduler/scheduler.go

```diff
@@ -450,6 +471,10 @@ func (s *Scheduler) forwardRequestToQuerier(querier schedulerpb.SchedulerForQuer
 	// Make sure to cancel request at the end to clean up resources.
 	defer s.cancelRequestAndRemoveFromPending(req.FrontendAddress, req.QueryID, "request complete")
 
+	queryComponentName := req.ExpectedQueryComponentName()
+	s.requestQueue.QueryComponentCapacity.IncrementForComponentName(queryComponentName)
```

Comment:
[nit] `QueryComponentCapacity.IncrementForComponentName` makes it sound like we're incrementing the capacity, but we're incrementing the inflight requests or utilisation. What do you think about renaming either `QueryComponentCapacity` or `IncrementForComponentName` to make this clearer? (eg. `IncrementInflightRequestsForComponentName`)

Reply:
renamed QueryComponentCapacity to `QueryComponentUtilization`, that way the utilization is what we are incrementing and decrementing.

Already had a bunch the "utilization" language in dosctrings and comments, I think it's fairly clear.